docs/src/pages/demos/breadcrumbs/CustomizedBreadcrumbs.tsx

```diff
@@ -9,52 +8,41 @@ import Avatar from '@material-ui/core/Avatar';
 import HomeIcon from '@material-ui/icons/Home';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 
-interface CustomBreadcrumbProps extends WithStyles<typeof styles> {}
+const StyledBreadcrumb = withStyles((theme: Theme) => ({
+  root: {
+    backgroundColor: theme.palette.grey[100],
+    height: 24,
+    color: theme.palette.grey[800],
+    fontWeight: theme.typography.fontWeightRegular,
+    '&:hover, &:focus': {
+      backgroundColor: theme.palette.grey[300],
+    },
+    '&:active': {
+      boxShadow: theme.shadows[1],
+      backgroundColor: emphasize(theme.palette.grey[300], 0.12),
+    },
+  },
+}))((props: any) => <Chip {...props} />);
```

Comment:
The last one, I don't know how to take care of it 😞.

Reply:
Yeah that's a nasty one: Microsoft/TypeScript#26591. I'll push a better fix (better because it reveals another type issue).

Reply:
Thank you!